### PR TITLE
Don't use ubi7/nodejs-8

### PR DESCRIPTION
**Pull request by rlcundiff**
Build with ubi7-minimal and install node.js on top.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -46,10 +46,66 @@ LABEL name="Application Navigator" \
...
+    FD3A5288F042B6850C66B31F09FE44734EB7990E \
+    71DCFD284A79C3B38668286BC97EC7A07EDE3FC1 \
+    DD8F2338BAE7501E3DD5AC78C273792F7D83545D \
+    C4F0DFFF4E8C1A8236409D08E73BC641CC11F4C8 \
+    B9AE9905FFD7803F25714661B63B535A4C206CA9 \
+    77984A986EBC2AA786BC0F66B01FBB92821C587A \
+    8FCCA13FEF1D0C2E91008E09770F7A9A5AE15600 \
+    4ED778F539E3634C779C87C6D7062848A1AB005C \
+    A48C2BEE680E841632CD4E44F07496B3EB3C1762 \
+    B9E2F5981AA6E0CD28160D9FF13993A75599653C \
```
Comment (kinueng) [inline Dockerfile]: Verified these keys are from https://github.com/nodejs/node#release-keys

File: Dockerfile [hunk 2]
```diff
@@ -46,10 +46,66 @@ LABEL name="Application Navigator" \
...
+  ; do \
+    gpg --batch --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys "$key" || \
+    gpg --batch --keyserver hkp://ipv4.pool.sks-keyservers.net --recv-keys "$key" || \
+    gpg --batch --keyserver hkp://pgp.mit.edu:80 --recv-keys "$key" ; \
+  done \
+  && curl -fsSLO --compressed "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-$ARCH.tar.xz" \
+  && curl -fsSLO --compressed "https://nodejs.org/dist/v$NODE_VERSION/SHASUMS256.txt.asc" \
+  && gpg --batch --decrypt --output SHASUMS256.txt SHASUMS256.txt.asc \
+  && grep " node-v$NODE_VERSION-linux-$ARCH.tar.xz\$" SHASUMS256.txt | sha256sum -c - \
```
Comment (kinueng) [inline Dockerfile]: Verified these are the steps to confirm the node download is signed.
https://github.com/nodejs/node#verifying-binaries

File: Dockerfile [hunk 3]
```diff
@@ -46,10 +46,66 @@ LABEL name="Application Navigator" \
...
+  && tar -xJf "node-v$NODE_VERSION-linux-$ARCH.tar.xz" -C /usr/local --strip-components=1 --no-same-owner \
+  && rm "node-v$NODE_VERSION-linux-$ARCH.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt \
+  && ln -s /usr/local/bin/node /usr/local/bin/nodejs \
+  && set -ex \
+  && for key in \
+    6A010C5166006599AA17F08146C2130DFD2497F5 \
```
Comment (kinueng) [inline Dockerfile]: Not sure how to verify the origins of this key, but I have seen many public docker images use this key to install `yarn`

**Comment by rlcundiff** [comment]
https://github.com/kappnav/Issues/issues/55

**Pull request merged**